fe/fe-core/src/main/java/org/apache/doris/catalog/TableProperty.java

```
@@ -166,11 +164,22 @@ public TableProperty buildDynamicProperty() {
 
     private TableProperty executeBuildDynamicProperty() {
         HashMap<String, String> dynamicPartitionProperties = new HashMap<>();
+        boolean valid = true;
         for (Map.Entry<String, String> entry : properties.entrySet()) {
-            if (entry.getKey().startsWith(DYNAMIC_PARTITION_PROPERTY_PREFIX)) {
+            if (entry.getKey().startsWith(DynamicPartitionProperty.DYNAMIC_PARTITION_PROPERTY_PREFIX)) {
                 dynamicPartitionProperties.put(entry.getKey(), entry.getValue());
+                if (!DynamicPartitionProperty.DYNAMIC_PARTITION_PROPERTIES.contains(entry.getKey())) {
+                    valid = false;
```

should throw exception or  ignore unknown property here.